src/main/proto/wfa/measurement/internal/kingdom/exchange_details.proto

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.measurement.internal.kingdom;
+
+option java_package = "org.wfanet.measurement.internal.kingdom";
+option java_multiple_files = true;
+
+// Stores metadata about an Exchange.
+message ExchangeDetails {
```

Comment:
Any particular reason you don't just put this in the same file with Exchange? Similar question for other message below.

Reply:
Our convention is to separate them. There are actually build targets that need the Details but not the entity itself.